geode-benchmarks/src/main/java/org/apache/geode/benchmark/parameters/GeodeProperties.java

```diff
@@ -48,13 +69,25 @@ public static Properties locatorProperties() {
 
   public static Properties clientProperties() {
     Properties properties = new Properties();
-    properties.setProperty(ConfigurationProperties.ENABLE_TIME_STATISTICS, "true");
-    properties.setProperty(ConfigurationProperties.LOG_LEVEL, "config");
-    properties.setProperty(ConfigurationProperties.STATISTIC_SAMPLING_ENABLED, "true");
-    properties.setProperty(ConfigurationProperties.MEMBER_TIMEOUT, "8000");
-    return properties;
 
+    properties.setProperty(ENABLE_TIME_STATISTICS, "true");
+    properties.setProperty(LOG_LEVEL, "config");
+    properties.setProperty(STATISTIC_SAMPLING_ENABLED, "true");
+    properties.setProperty(MEMBER_TIMEOUT, "8000");
+
+    return properties;
   }
 
 
+  public static Properties withSsl(Properties properties) {
+    properties.setProperty(SSL_ENABLED_COMPONENTS, ALL);
+
+    // TODO make configurable, relative, something?
```

Comment:
It might be better to remove this here and write a chore to put on our backlog instead. TODOs in the code don't tend to get prioritized.

Reply:
Good catch, I removed the comment. The story is already created: https://www.pivotaltracker.com/story/show/167072556